Cargo.toml

```diff
@@ -26,4 +26,5 @@ once_cell = "1"
 [dev-dependencies]
 rustversion = "1"
 trybuild = "1"
-yew = "0.21.0"
+yew = { git = "https://github.com/yewstack/yew.git" }
+implicit-clone = { version = "0.4.9" }
```

Comment:
I don't think there is a way to reëxport? better add this to the Readme as a dependency.